src/deparam.js

```diff
@@ -121,7 +121,7 @@ function complex(key, value, obj, doCoerce) {
         } else if (nextProp) {
             const { ob, push } = resolveObj(obj[prop], nextProp);
             obj[prop] = ob;
-            const nextOb = push ? {} : obj[prop];
+            const nextOb = push ? Object.create(null) : obj[prop];
```

Comment:
@scssyworks this one you also reverted. I don’t understand why. But I also don’t understand whether it makes a difference or not…

Reply:
This is also vulnerable:

```javascript
deparam('test[a]=b&test[a][bla]=c&test[a][1][__proto__][test]=yay'); // Will put test: yay on the prototype
```

Reply:
Once again apologies on the merge error